spec/cli/commands/ip_spec.rb

```diff
@@ -5,62 +5,119 @@
 describe Ronin::CLI::Commands::Ip do
   include_examples "man_page"
 
-  let(:ipcmd) { Ronin::CLI::Commands::Ip.new}
+  subject { Ronin::CLI::Commands::Ip.new }
   let(:localhost) { "127.0.0.1" }
 
-  describe "-r" do
-    it "will reverse an IPv4 address" do
-      ipcmd.options[:reverse] = true
-      expect{ ipcmd.process_value(localhost) }.to output("1.0.0.127.in-addr.arpa\n").to_stdout
+  describe "#process_value" do
+
```

Comment:
Extra newline. Although not sure if you're going to add `before { }` stuff here in the next commit?

Reply:
I had considered it but we'll see where the specs land on the next iteration.  If it's too noisy, I can convert this back to a draft.